### PR TITLE
Train 112

### PR DESCRIPTION
**Pull request by spahl**
#1070 
#1065 
#1069 
#1067

### CODE REVIEW

**Comment by spahl** [comment]
:ship:

File: packages/dcos-ui/buildinfo.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-ui.git",
-    "ref": "ffec1877fbfd15e2f44c045d60059107782c6dd2",
+    "ref": "1d300f156c88195d60c87a643d83f1f301bfe8a8",
```
Comment (rcorral) [inline packages/dcos-ui/buildinfo.json]: This needs to be `0cbd3f5d204d5f57b0c0893417364f7377426068`

Comment (spahl) [inline packages/dcos-ui/buildinfo.json]: Too late:-) Create a new PR:-)

**Comment by spahl** [comment]
We finally found what makes the signal service flaky. To be adressed here: https://mesosphere.atlassian.net/browse/DCOS-12329

**Pull request merged**